lib/Conversion/LLHDToLLVM/LLHDToLLVM.cpp

```diff
@@ -1907,21 +1906,10 @@ void LLHDToLLVMLoweringPass::runOnOperation() {
   populateCombToArithConversionPatterns(converter, patterns);
   arith::populateArithToLLVMConversionPatterns(converter, patterns);
 
-  target.addLegalDialect<LLVM::LLVMDialect>();
   target.addLegalOp<ModuleOp>();
 
-  // Apply a full conversion to remove unrealized conversion casts.
   if (failed(applyFullConversion(getOperation(), target, std::move(patterns))))
```

Comment:
IMO this should be a partial conversion (and the LLHD dialect ought to be marked illegal), so that this pass doesn't choke on unknown ops.

Reply:
This pass needs to be refactored anyway, so anything that makes the tests succeed is fine for now.